src/core/core.cpp

```diff
@@ -5,7 +5,7 @@
 #include <vector>
 using json = nlohmann::json;
 
-std::vector<Product> getProducts(const json& jsonData) {
+std::vector<Product> const getProducts(const json& jsonData) {
```

Comment:
this also is `const` data return, 
`const` method declaration looks like this:
return_type method_name (arguments) `const`;